### PR TITLE
GCW-2415 fix UI for browse messaging

### PR DESCRIPTION
**Pull request by abhinav098**
Hello Team, 
This PR fixes UI issues on browse app for messaging.
Thanks

### CODE REVIEW

File: app/components/variable-height-textarea.js [hunk 1]
```diff
@@ -5,18 +5,28 @@ export default Ember.TextArea.extend({
   attributeBindings: ["disabled"],
   disabled: false,
 
+   didDestroyElement: function () {
+     Ember.$('.liquid-container').css({
```
Comment (patrixr) [inline app/components/variable-height-textarea.js]: is liquid container an ember thing ?

Comment (patrixr) [reply app/components/variable-height-textarea.js]: `Ember.$('.liquid-container')` will fetch any liquid container across the whole app. Which sounds dangerous.
If this is really needed, make sure to only apply the changes on the liquid-container that matter (i'm guessing the one containing the textarea)

Comment (abhinav098) [reply app/components/variable-height-textarea.js]: `liquid-container` is just a class defined to the main `div`, after body on browse. the CSS which I'm applying at the time of destroyComponent is already defined. All I'm doing is unsetting at the time of component render, and resetting it back as soon as the component goes away.

Comment (abhinav098) [reply app/components/variable-height-textarea.js]: @patrixr i did not want to modify that thing globally so i took this approach of unsetting and resetting by using component lifecycle methods.

Comment (Darshan-Chauhan) [inline app/components/variable-height-textarea.js]: Should we add a check if liquid container exists or not?

File: app/components/variable-height-textarea.js [hunk 2]
```diff
@@ -5,18 +5,28 @@ export default Ember.TextArea.extend({
...
+       "position": "relative",
+       "-webkit-transform": "translateY(0)",
+       "-moz-transform": "translateY(0)",
+       "transform": "translateY(0)"
+     });
+   },
+
   didInsertElement: function () {
+    Ember.$('.liquid-container').css({
```
Comment (Darshan-Chauhan) [inline app/components/variable-height-textarea.js]: ^

File: app/controllers/orders/conversation.js [hunk 3]
```diff
@@ -20,9 +20,14 @@ export default detail.extend({
   sortedMessages: Ember.computed.sort("model.messages", "sortProperties"),
 
   groupedMessages: Ember.computed("sortedMessages", function () {
+    this.autoScroll();
     return this.groupBy(this.get("sortedMessages"), "createdDate");
   }),
 
+  autoScroll: function () {
```
Comment (Darshan-Chauhan) [inline app/controllers/orders/conversation.js]: can we rename as `autoScroll()`?

File: app/components/variable-height-textarea.js (hunk 1 above)
Comment (abhinav098) [reply app/components/variable-height-textarea.js]: It's always there. But I'll add a check if you want.

File: app/controllers/orders/conversation.js (hunk 3 above)
Comment (abhinav098) [reply app/controllers/orders/conversation.js]: yeah sure.

**Comment by namrataukirde** [comment]
:+1:

**Comment by namrataukirde** [comment]
:+1:

**Pull request merged**